backend/api/chat_orchestrator.py

```
@@ -30,7 +30,9 @@
     from services.scene_memory import scene_memory_manager
     LANGCHAIN_AVAILABLE = True
 except ImportError as e:
```

Ruff (0.14.1)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)